src/ConnectedMode.UnitTests/BranchMatcherTests.cs

```diff
@@ -325,15 +351,15 @@ private static Mock<ISonarQubeService> CreateSonarQubeServiceWithTypes(string ma
             var service = new Mock<ISonarQubeService>();
             service.Setup(x => x.IsConnected).Returns(true);
 
-            IList<SonarQubeProjectBranch> remoteBranches = new List<SonarQubeProjectBranch>
-            {
-                new SonarQubeProjectBranch(mainBranch, true, DateTime.Now, "BRANCH")
-            };
+            IList<SonarQubeProjectBranch> remoteBranches = new List<SonarQubeProjectBranch>();
 
             foreach (var branch in branches)
             {
-                remoteBranches.Add(new SonarQubeProjectBranch(branch.branchName, true, DateTime.Now, branch.type));
+                remoteBranches.Add(new SonarQubeProjectBranch(branch.branchName, false, DateTime.Now, branch.type));
```

Comment:
We had a bug in our tests (the product code was correct), it only affected the test "GetMatchedBranch_NoMatchingBranch_ChooseMain" and it worked before the master was the first. The test started to fail after moving the master.